crt_portal/cts_forms/validators.py

```diff
@@ -0,0 +1,40 @@
+import logging
+import requests
+
+from django.conf import settings
+from django.core.exceptions import ValidationError
+
+logger = logging.getLogger(__name__)
+
+# https://github.com/ajilaag/clamav-rest#status-codes
+AV_SCAN_CODES = {
+    'CLEAN': [200],
+    'INFECTED': [406],
+    'ERROR': [400, 412, 500, 501],
+}
+
+
+def _scan_file(file):
+    return requests.post(settings.AV_SCAN_URL, files={'file': file}, data={'name': file.name})
+
+
+def validate_file_infection(file):
```

Comment:
In testing locally I haven't been able to get this to fail on an upload in excess of 100mb.

The clamav container reports: `^INSTREAM: Size limit reached, (requested: 1024, max: 502)` but the python app isn't responding to the browser request. 

This may be a non-issue once the other, application side, file checks are in place and we're returning an error message on large files before handing to clamav.

Reply:
Yes- @ilodidoj 's validation work has been pulled in, so file size is now checked before anything gets sent to ClamAV:

![image](https://user-images.githubusercontent.com/1425377/108559433-f08d2000-72c0-11eb-8ced-9aa813e288b6.png)